docs/paper.md

```diff
@@ -50,57 +50,61 @@ affiliations:
 
 `scores` not only includes common scores (e.g. Mean Absolute Error), it also includes novel scores not commonly found elsewhere (e.g. FIxed Risk Multicategorical (FIRM) score, Flip-Flop Index), complex scores (e.g. threshold-weighted continuous ranked probability score), and statistical tests (such as the Diebold Mariano test). It also contains isotonic regression which is becoming an increasingly important tool in forecast verification and can be used to generate stable reliability diagrams. Additionally, it provides pre-processing tools for preparing data for scores in a variety of formats including cumulative distribution functions (CDF). At the time of writing, `scores` includes over 50 metrics, statistical techniques and data processing tools.
 
 All of the scores and statistical techniques in this package have undergone a thorough scientific and software review. Every score has a companion Jupyter Notebook tutorial that demonstrates its use in practice.
 
-`scores` primarily supports `xarray` datatypes for Earth system data, allowing it to work with NetCDF4, HDF5, Zarr and GRIB data sources among others. `scores` uses Dask for scaling and performance. It has expanding support for `pandas`.  
+`scores` primarily supports `xarray` datatypes for Earth system data, allowing it to work with NetCDF4, HDF5, Zarr and GRIB data sources among others. `scores` uses Dask for scaling and performance. It has expanding support for `pandas`.
 
-The software repository can be found at [https://github.com/nci/scores/](https://github.com/nci/scores/).
+The `scores` software repository can be found at [https://github.com/nci/scores/](https://github.com/nci/scores/).
 
 \pagebreak
 
 # Statement of Need
 
-The purpose of this software is (a) to mathematically verify and validate models and predictions and (b) to foster research into new scores and metrics. 
+Labelled, n-dimensional data is widely used in many scientific fields. The Earth system science community makes heavy use of physics-based and machine learning models, both to process observations (such as identifying land use from satellite data) and to make predictions about the future (such as forecasting the weather). These models, predictions and forecasts undergo verification and evaluation to assess their correctness. 
+
+The purpose of `scores` is (a) to mathematically verify and validate models and predictions and (b) to foster research into new scores and metrics. 
+
+`scores` handles dimensionality and weighting (e.g. latitude weighting) more effectively than commonly-used data science packages. While there are existing open source Python verification packages for labelled n-dimensional data (see "Related Software Packages" further below), none of these packages offer all of the key benefits of `scores`.
```

Comment:
I wasn't sure whether the end of the first sentence should mention some of the commonly-used data science packages being referred to. Eg:
"scores handles dimensionality and weighting (e.g. latitude weighting) more effectively than commonly-used data science packages (e.g. SciPy (Ref); scilit-learn (Ref)). While there are existing..."

If so, for the comparison with the commonly-used data science packages, perhaps that would be sufficient? And the "Related Software Packages" section, with the more detailed breakdown, could perhaps remain for the packages specifically for verifying labelled n-dimensional data?

Reply:
My opinion is that scikit-learn and scipy are not relevant comparisons.

I don't think general data science applications should use scores, when they can use scikit-learn or scipy, because scores isn't a general data science package.

Other labelled n-dimensional packages are a stronger comparisons to scikit-learn and scipy.

Furthermore, for machine learning applications specific to labelled n-dimensional data in the geoscience communities, then scikit-learn for example starts to become less desirable, due to its limitations; and the other packages are better comparisons anyway.

Reply:
I agree that scikit-learn and scipy are not close alternatives for scores. However, for readers coming from ML or data science of that kind, it may not be apparent what differentiates scores and why you would bother. It would be great if we had a short, clear statement which was easily understood by "scipy people" which would frame scores appropriately. I just can't think of one. This would also respond to the specific reviewer feedback, but also this resonates with some of our internal feedback, so I think we do need to touch on it.

Reply:
Perhaps something like "Compared to many popular data science libraries, scores works more effectively with higher-dimensional structures where a complex MultiIndex table might otherwise be necessary, or to preserve a more natural coordinate-based indexing approach to accessing information". 

Or, maybe, the current sentence is fine.

Reply:
I think the current sentence is fine, except "more effectively" maybe isn't specific enough?

Reply:
In case people land on including citations to SciPy and/or scikit-learn, I am going to paste the BibTeX here. If it isn't needed, no worries. But if it is needed, it will save anyone from having to find this information again. 

Note: if the SciPy reference is included, will need to check the author names render properly in the JOSS PDF. They've been pretty careful with including braces (where applicable) in author names to promote proper rendering, so I think there is a good chance it will render correctly. But the final rendering in the JOSS PDF should still be checked.

@ARTICLE{Virtanen:2020,
  author  = {Virtanen, Pauli and Gommers, Ralf and Oliphant, Travis E. and
            Haberland, Matt and Reddy, Tyler and Cournapeau, David and
            Burovski, Evgeni and Peterson, Pearu and Weckesser, Warren and
            Bright, Jonathan and {van der Walt}, St{\'e}fan J. and
            Brett, Matthew and Wilson, Joshua and Millman, K. Jarrod and
            Mayorov, Nikolay and Nelson, Andrew R. J. and Jones, Eric and
            Kern, Robert and Larson, Eric and Carey, C J and
            Polat, {\.I}lhan and Feng, Yu and Moore, Eric W. and
            {VanderPlas}, Jake and Laxalde, Denis and Perktold, Josef and
            Cimrman, Robert and Henriksen, Ian and Quintero, E. A. and
            Harris, Charles R. and Archibald, Anne M. and
            Ribeiro, Ant{\^o}nio H. and Pedregosa, Fabian and
            {van Mulbregt}, Paul and {SciPy 1.0 Contributors}},
  title   = {{SciPy} 1.0: Fundamental Algorithms for Scientific
            Computing in Python},
  journal = {Nature Methods},
  year    = {2020},
  volume  = {17},
  pages   = {261--272},
  adsurl  = {https://rdcu.be/b08Wh},
  doi     = {10.1038/s41592-019-0686-2},
}

Note: I couldn't find any DOI information for Journal of Machine Learning Research articles.

@article{Pedregosa:2011,
  author  = {Fabian Pedregosa and Ga{{\"e}}l Varoquaux and Alexandre Gramfort and Vincent Michel and Bertrand Thirion and Olivier Grisel and Mathieu Blondel and Peter Prettenhofer and Ron Weiss and Vincent Dubourg and Jake Vanderplas and Alexandre Passos and David Cournapeau and Matthieu Brucher and Matthieu Perrot and {{\'E}}douard Duchesnay},
  title   = {Scikit-learn: Machine Learning in Python},
  journal = {Journal of Machine Learning Research},
  year    = {2011},
  volume  = {12},
  number  = {85},
  pages   = {2825-2830},
  url     = {http://jmlr.org/papers/v12/pedregosa11a.html}
}

Reply:
I think that the sentence in this pull request is fine as is and I don't think we need to compare to SciPy and scikit-learn. They don't focus on n-dimensional labelled data.

Reply:
I think people are sufficiently agreed that the current text is fine. I am going to resolve this conversation. I will reflect a little, and if I can think of anything more specific to say than "more effectively" for the dimension handling, I will raise that as a new comment. Thanks all.